### ISSUE TITLE
How to adapt when the dataset of the target domain is much smaller than source domain

### ISSUE THREAD

**Issue by universe27king** [corenel/pytorch-adda#30]
eg.source domain has more than 8000 samples while target domain has about 200 samples.
ACC decreases after adaptation, Reduce the learning rate and adapt epochs seems to be ineffective
